internal/setup/setup_test.go

```diff
@@ -39,7 +38,7 @@ var setupTests = []setupTest{{
 	summary: "Missing archives",
 	input: map[string]string{
 		"chisel.yaml": `
-			format: chisel-v1
```

Comment:
I think we should have a test with `chisel-v1` failing because it is `chisel-v1` (not because of the rest of the format).

Reply:
Done in https://github.com/letFunny/chisel/pull/16/commits/ecf93be81e5c7acacb6493cbd2ae83b032ea89b8.